resources/install.rb

```diff
@@ -137,8 +137,10 @@
                 restart_mode: 'always',
                 after_target: 'network',
                 auto_reload: true,
+                pid_file: '/run/haproxy.pid',
+                conf_file: new_resource.config_file,
+                bin_prefix: new_resource.bin_prefix,
                 template: 'haproxy:haproxy.service.erb'
-        action :nothing
```

Comment:
IIRC we need these to just setup the service in the Chef resource collection.

Reply:
Without this unit file will not get updated even with user changes unless, say, HAProxy config is updated, as in https://github.com/sous-chefs/haproxy/blob/master/resources/install.rb#L121-L124 . It doesn't look that harmful to me as the service will not be updated / restarted unless there is a need for it to be.

Reply:
That's there to stop the main template blatting the accumulator pattern. Not particularly intuitive if you've not seen that before. 

For now we'll leave that alone and just pull in the bin-prefix stuff if thats ok?